test.js

```diff
@@ -19,8 +19,5 @@ test('ignores control characters', t => {
 	t.is(m(String.fromCharCode(134)), 0);
 	t.is(m(String.fromCharCode(159)), 0);
 	t.is(m('\u001B'), 0);
-});
-
-test.failing('handles combining characters', t => {
```

Comment:
Maybe keep this test in case of a regression in the future so we can pinpoint it? Just remove the `.failing` part.